CONTRIBUTING.md

```diff
@@ -2,43 +2,88 @@
+     |--------------------------------------------------->|                                                                                  |
+     |                                                    |                                                                                  |
+     |                                                    | Authenticate and request Conjur access token                                     |
+     |                                                    |--------------------------------------------------------------------------------->|
+     |                                                    |                                                                                  |
+     |                                                    |      Return Conjur access token encrypted by Conjur server certificate, or error |
+     |                                                    |<---------------------------------------------------------------------------------|
+     |                                                    |                                                                                  |
+     |     Return Conjur access token encrypted over mTLS |                                                                                  |
+     |<---------------------------------------------------|                                                                                  |
+     |                                                    |                                                                                  |
+     | Request secret values using token                  |                                                                                  |
+     |-------------------------------------------------------------------------------------------------------------------------------------->|
+     |                                                    |                                                                                  |
+     |                                                    |                                                   Return secret values, or error |
+     |<--------------------------------------------------------------------------------------------------------------------------------------|
+     |                                                    |                                                                                  |
+     | Return secret values, or error                     |                                                                                  |
+     |--------------------------------------------------->|                                                                                  |
+     |                                                    |                                                                                  |
+```
 
-# Running services locally
+This was generated by https://textart.io/sequence using the sequence:
```

Comment:
This should be in a collapsed section I think - it's somewhat irrelevant if we include the pic

Reply:
it's quite brief, so I'm ok with leaving it here rather than adding collapsed sections yet